### PR TITLE
GG-18500 Possibility to enable loading of example schemas for Oracle database.

### PR DESCRIPTION
**Pull request by vsisko**

### CODE REVIEW

**Comment by vsisko** [comment]
We show that flag only for Oracle database.

File: modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/handlers/DatabaseHandler.java [hunk 1]
```diff
@@ -172,7 +172,7 @@ public DbSchema collectDbSchemas(WebSocketEvent evt) throws SQLException {
                 catalog = parts.length > 0 ? parts[parts.length - 1] : "NONE";
             }
 
-            Collection<String> schemas = dbMetaReader.schemas(conn);
+            Collection<String> schemas = dbMetaReader.schemas(conn, args.getBoolean("importSamples"));
```
Comment (akuznetsov-gridgain) [inline modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/handlers/DatabaseHandler.java]: Lets use args.getBoolean("importSamples", false) ?

Comment (vsisko) [reply modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/handlers/DatabaseHandler.java]: Added default argument value.

File: modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/db/dialect/MySQLMetadataDialect.java [hunk 2]
```diff
@@ -39,12 +39,13 @@
     }
 
     /** {@inheritDoc} */
-    @Override public Collection<String> schemas(Connection conn) throws SQLException {
+    @Override public Collection<String> schemas(Connection conn, boolean importSamples) throws SQLException {
         Collection<String> schemas = new ArrayList<>();
 
         ResultSet rs = conn.getMetaData().getCatalogs();
```
Comment (akuznetsov-gridgain) [inline modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/db/dialect/MySQLMetadataDialect.java]: The only difference between  MySQLMetadataDialect  and JdbcMetadataDialect is this line:
JdbcMetadataDialect: ResultSet rs = conn.getMetaData().getSchemas();
MySQLMetadataDialect : ResultSet rs = conn.getMetaData().getCatalogs();

Almost the same for OracleMetadataDialect.

May be it make sense to refactor as method that will return collection strings with schemas names?

Comment (vsisko) [reply modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/db/dialect/MySQLMetadataDialect.java]: Removed code duplication on collection of schemas.

**Pull request merged**